bundles/org.eclipse.ui.win32/.project

```diff
@@ -3,13 +3,6 @@
 	<name>org.eclipse.ui.win32</name>
 	<comment></comment>
 	<projects>
-		<project>org.eclipse.core.boot</project>
-		<project>org.eclipse.core.resources</project>
-		<project>org.eclipse.core.runtime</project>
-		<project>org.eclipse.help</project>
-		<project>org.eclipse.ui</project>
-		<project>org.eclipse.ui.ide</project>
-		<project>org.eclipse.ui.views</project>
```

Comment:
Does anybody know why this was here? Besides `org.eclipse.core.boot` all of those Plug-ins are dependencies of this one and are therefore are implicitly required by the PDE build order computer.
I have never seen `org.eclipse.core.boot` before and assume it exists in pre OSGi times?

When searching the entire platform-aggregator tree I only see some hits in the documentation, e.g. in the Platform class referring to `org.eclipse.core.boot.BootLoader`. It looks like that is obsolete as well?

Reply:
@mickaelistria, @akurtakov, @iloveeclipse can you tell anything about that?

Reply:
> I have never seen org.eclipse.core.boot before and assume it exists in pre OSGi times?

Same here. This is from Eclipse 2.0 and can be removed.

Reply:
OK. Thanks.